cmd/http.go

```diff
@@ -89,14 +89,20 @@ Examples:
   # HTTP HEAD request to jsdelivr.com from 2 probes in New York (protocol, port and path are inferred from the URL)
   http https://www.jsdelivr.com:443/package/npm/test?nav=stats from New York --limit 2
```

Comment:
@didil Can you locally test if this example works properly?

Reply:
@jimaek just did, it's working. Did you run into issues with this specific example ?